.changeset/quick-months-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"focus-trap-react": minor
+---
+
```

Comment:
What about mentioning `setReturnFocus` as well?

Reply:
Missed this before I merged. Yes, that's a good idea. I'll add it to the changeset, thank you!